src/index.html

```diff
@@ -8,6 +8,49 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+
+    <div class="star stars--0">
```

Comment:
```suggestion
    <div class="stars stars--0">
```
bem block should have the same name as beginning of bem elements